openreview/venue_request/venue_request.py

```diff
@@ -185,6 +185,81 @@ def setup_review_stage(self):
             }
         ))
 
+    def setup_official_comment_stage(self):
+
+        official_comment_stage_content = {
+            'commentary_start_date': {
+                'description': 'When does official commentary begin? Please use the following format: YYYY/MM/DD HH:MM (e.g. 2019/01/31 23:59)',
+                'value-regex': r'^[0-9]{4}\/([1-9]|0[1-9]|1[0-2])\/([1-9]|0[1-9]|[1-2][0-9]|3[0-1])(\s+)?((2[0-3]|[01][0-9]|[0-9]):[0-5][0-9])?(\s+)?$',
+                'order': 27
+            },
+            'allow_unsubmitted_reviewers_to_comment': {
+                'description': 'Should the reviewers who have not submitted their review be able to make official comments? Default is "No, unsubmitted reviewers should not be able to post official comments"',
+                'value-radio': [
+                    'Yes, unsubmitted reviewers should be able to post official comments',
+                    'No, unsubmitted reviewers should not be able to post official comments'
+                ],
+                'required': True,
+                'default': 'No, unsubmitted reviewers should not be able to post official comments',
+                'order': 28
+            },
+            'allow_reader_selection': {
+                'description': 'Should the comment author be allowed to select comment readers from a list? Default is "No, comment authors should not be able to select comment readers"',
+                'value-radio': [
+                    'Yes, comment authors should be able to select comment readers',
+                    'No, comment authors should not be able to select comment readers'
+                ],
+                'required': True,
+                'default': 'No, comment authors should not be able to select comment readers',
+                'order': 29
+            },
+            'email_program_chairs_about_official_reviews': {
+                'description': 'Should the PCs receive an email for each official comment made in the venue? Default is "No, do not email PCs for each official comment in the venue"',
+                'value-radio': [
+                    'Yes, email PCs for each official comment made in the venue',
+                    'No, do not email PCs for each official comment made in the venue'
+                ],
+                'required': True,
+                'default': 'No, do not email PCs for each official comment made in the venue',
+                'order': 30
+            },
+            'allow_authors_to_comment': {
```

Comment:
I think `allow_authors_to_comment` and `allow_unsubmitted_reviewers_to_comment` should be together in a single field where the users can multi-select the participants of the discussions:

- public
- authors
- all reviewers
- assign reviewers
- submitted reviewers
- area chairs
- program chairs

I think if public is selected then we can create the public comments invitations. I'm not totally convinced about this, thoughts @mohituniyal @celestemartinez ?

Reply:
Changes have been made for all other comments. About this one, I like the idea of aggregating all the separate readership questions into 1. The field name and description would be very important though.

Reply:
this change that Mohit is saying is not implemented, I can not find it. @celestemartinez can you work on this?